src/app/content/types.ts

```diff
@@ -44,6 +44,7 @@ export interface BookWithOSWebData extends ArchiveBook {
   theme: 'blue' | 'green' | 'gray' | 'yellow' | 'deep-green' | 'light-blue' | 'orange' | 'red';
   slug: string;
   publish_date: string;
+  amazon_link?: string;
```

Comment:
is this really potentially missing? i thought it would always be there but might be `''`